lang/tr.json

```diff
@@ -118,30 +118,30 @@
           "togglePlayerSounds": "Oyuncu Sesleri",
           "toggleEnableExplorer": {
             "label": "Yume 2kki Gezginini",
-            "helpText": "Yume 2kki Gezgini konumunuzun bağlantılarını tek bir yerde görmenize izin veren bir Yume 2kki Kaşifi entegrasyonudur. Bu özellik deneyiminizi derinlemesine etkiler ve bu özelliği etkinleştirmeden önce oyunun çoğu kısmını bilgisizce keşfetmeniz önerilir. Yume 2kki Gezginini kullanmak için hesabınıza giriş yapmanız gereklidir."
```

Comment:
This was not an error, this line references Yume 2kki Explorer for this specific part and not Yume 2kki Navigator. Yume 2kki Explorer and Yume 2kki Navigator are different things: the Explorer is a website that shows you a graph of the different 2kki worlds (can be viewed here: https://2kki.app), while the Navigator is a feature embed in YNOproject that allows you to view the worlds near where you are and tracks your world completion, using the same type of graph.